controlplane/kubeadm/internal/machinefilters/machine_filters.go

```diff
@@ -23,10 +23,10 @@ import (
 	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha3"
 )
 
-type MachineFilter func(machine *clusterv1.Machine) bool
+type Func func(machine *clusterv1.Machine) bool
 
 // And returns a MachineFilter function that returns true if all of the given MachineFilters returns true
```

Comment:
All of these comments should probably be updated to s/MachineFilter/machinefilter.Func/